src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -21,16 +32,40 @@ public TimelineViewModel()
             this.WhenAnyValue(x => x.SelectedDate).Subscribe(HandleSelectedDateChanged);
             SelectPreviousDay = ReactiveCommand.Create(Toggl.ViewTimelinePreviousDay);
             SelectNextDay = ReactiveCommand.Create(Toggl.ViewTimelineNextDay);
+            IncreaseScale = ReactiveCommand.Create(() => ChangeScale(-1));
+            DecreaseScale = ReactiveCommand.Create(() => ChangeScale(1));
+            this.WhenAnyValue(x => x.SelectedScaleMode).Subscribe(_ =>
+                HourHeightView = _scaleModes[SelectedScaleMode] * GetHoursInLine(SelectedScaleMode));
             Toggl.OnTimeline += HandleDisplayTimeline;
-            Toggl.OnTimeEntryList += HandleTimeEntryListChanged; 
+            Toggl.OnTimeEntryList += HandleTimeEntryListChanged;
+            UpdateHourViews();
+        }
+
+        private void ChangeScale(int value)
+        {
+            var newSelectedScaleMode = SelectedScaleMode + value < 0 || SelectedScaleMode + value > _scaleModes.Count - 1
+                ? SelectedScaleMode
+                : SelectedScaleMode + value;
+            ScaleRatio = 1.0*_scaleModes[newSelectedScaleMode] / _scaleModes[SelectedScaleMode];
+            SelectedScaleMode = newSelectedScaleMode;
+            HandleScaleChanged(SelectedScaleMode);

```

Comment:
This is not a big deal, but I'd like to not drift away from the ReactiveUI way of doing things - writing more declarative code instead of procedural.
E.g. 
- `HandleScaleChanged` - the method could be set up as a subscription to the changed value of `SelectedScaleMode`, the name of the method doesn't describe "what" is being done there.
- `ScaleRatio` - can be a computed property
- `HourHeightView` - can be a computed property
etc.

The principles are described here https://www.reactiveui.net/docs/handbook/view-models/